### PR TITLE
new chapter

### PR DESCRIPTION
**Pull request by TS96**
Please review this, @casey-collab

### CODE REVIEW

File: english/quit/quit.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+You're trying really hard to understand this,
```
Comment (casey-collab) [inline english/quit/quit.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/quit/quit.md (hunk 1 above)
Comment (TS96) [inline english/quit/quit.md]: Please review this, @casey-collab

**Comment by TS96** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :night_with_stars:

**Pull request merged**